README.md

```diff
@@ -291,7 +291,7 @@ public struct MyApp: App, Instantiable {
 }
 ```
 
-An `Instantiator` is `@MainActor`-bound – if you want to instantiate a type in a nonisolated environment, use a [`NonisolatedInstantiator`](Sources/SafeDI/DelayedInstantiation/NonisolatedInstantiator.swift).
+An `Instantiator` is not `Sendable` – if you want to instantiate a type on a different queue than the one used by your root object, use a [`SendableInstantiator`](Sources/SafeDI/DelayedInstantiation/SendableInstantiator.swift).
```

Comment:
I think people reading this will understand, but is using the word "queue" here incorrect? In the high level docs, Apple doesn't refer to these things as "queues" but "concurrent contexts" or "actors."

Reply:
solid feedback. How about:
```suggestion
An `Instantiator` is not `Sendable` – if you want to be able to share an instantiator across concurrency domains, use a [`SendableInstantiator`](Sources/SafeDI/DelayedInstantiation/SendableInstantiator.swift).
```

General language stolen from the [Swift programming book](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/concurrency/):

<img width="884" alt="Some kinds of data can’t be shared between concurrency domains, because that data contains mutable state, but it doesn’t protect against overlapping access" src="https://github.com/dfed/SafeDI/assets/139364/f971934d-607c-4de3-873e-eb4882fce5db">